dartagnan/src/main/java/com/dat3m/dartagnan/program/analysis/alias/FieldSensitiveAndersen.java

```diff
@@ -34,44 +34,31 @@
  * Any structure that occurs
  * <p>
  * Structures, that never occurs in any expression, are considered unreachable.
- *
- * @author xeren
  */
 public class FieldSensitiveAndersen implements AliasAnalysis {
 
     ///When a pointer set gains new content, it is added to this queue
-    private final Queue<Object> variables = new ArrayDeque<>();
+    private final LinkedHashSet<Object> variables = new LinkedHashSet<>();
```

Comment:
Duplicate elimination, since the queue does not contain any deltas to propagate, but instead propagates the entire location set.  `LinkedHashSet` still implements FIFO, so the comment remains valid.